client/css/base/_tables.css

```diff
@@ -1,20 +1,33 @@
 table {
-    overflow: hidden;
+  overflow: hidden;
+  width: 100%;
+  overflow-x: auto;
+  display: block;
 }
 
-td, th {
-    border: 1px solid #DADFE0;
-    padding: 5px;
+td,
+th {
+  width: 400px;
+  border-bottom: 1px solid $light-grey;
+  padding: 5px;
 }
 
 th {
-    text-align: center;
+  text-align: left;
 }
 
 th.left {
-    text-align: left;
+  text-align: left;
```

Comment:
not sure where this class is used actualy. I did not touched profile pages, maybe you are refereing that? I can remove anyway, no problem.